### PR TITLE
fix: kernel and simulate ci runs

### PR DESCRIPTION
**Pull request by highskore**

### CODE REVIEW

File: src/test/ModuleKitHelpers.sol [hunk 1]
```diff
@@ -837,6 +838,27 @@ library ModuleKitHelpers {
...
 
+    /// @dev Kernel requires us to temporarily disable the hook multiplexer to use smart sessions
+    modifier withHookFixForKernel(AccountInstance memory instance) {
+        // Check if account is KERNEL
+        if (instance.accountType == AccountType.KERNEL) {
+            // Cache hook multiplexer
+            address hookMultiplexer =
+                KernelHelpers(instance.accountHelper).getHookMultiPlexer(instance);
+            // Uninstall MockHookMultiplexer
+            instance.uninstallModule(MODULE_TYPE_HOOK, hookMultiplexer, "");
```
Comment (kopy-kat) [inline src/test/ModuleKitHelpers.sol:849]: I think this is not needed

File: src/test/ModuleKitHelpers.sol [hunk 2]
```diff
@@ -837,6 +838,27 @@ library ModuleKitHelpers {
...
+            // Set hook multiplexer to address(1)
+            KernelHelpers(instance.accountHelper).setHookMultiPlexer(instance, address(1));
+            _;
+            // Set hook multiplexer back to MockHookMultiplexer
+            KernelHelpers(instance.accountHelper).setHookMultiPlexer(instance, hookMultiplexer);
+            // Reinstall MockHookMultiplexer
+            instance.installModule(MODULE_TYPE_HOOK, hookMultiplexer, "");
```
Comment (kopy-kat) [inline src/test/ModuleKitHelpers.sol:856]: this as well

File: package.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
     "@ERC4337/account-abstraction": "github:kopy-kat/account-abstraction#develop",
     "@ERC4337/account-abstraction-v0.6": "github:eth-infinitism/account-abstraction#v0.6.0",
     "@prb/math": "^4.0.2",
-    "@rhinestone/erc4337-validation": "^0.0.3",
```
Comment (kopy-kat) [inline package.json:33]: if the foundry pr is merged should we try again with latest?

**Review by kopy-kat — APPROVED** [review]

File: src/test/ModuleKitHelpers.sol (hunk 1 above)
Comment (highskore) [reply src/test/ModuleKitHelpers.sol:849]: Actually it reverts if we don't uninstall it, not sure about having to re-install it (doesn't revert if i remove it) but it made sense to re-install after runinstalling it.

**Pull request merged**